x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/persistence/ProfilingDataStreamManager.java

```diff
@@ -141,8 +141,11 @@ public void onFailure(Exception e) {
     private void createDataStream(ProfilingDataStream dataStream, final ActionListener<? super ActionResponse> listener) {
         final Executor executor = threadPool.generic();
         executor.execute(() -> {
-            CreateDataStreamAction.Request request = new CreateDataStreamAction.Request(dataStream.getName());
-            request.masterNodeTimeout(TimeValue.timeValueMinutes(1));
+            CreateDataStreamAction.Request request = new CreateDataStreamAction.Request(
+                TimeValue.ONE_MINUTE /* TODO should we wait longer? */,
+                TimeValue.THIRTY_SECONDS /* TODO should we wait longer? */,
```

Comment:
Unsure about these timeouts tbh, I'm leaving them as-is but at least calling them out as such here.